cypress/integration/universityCanSeeRegistrationKeys.spec.js

```diff
@@ -0,0 +1,40 @@
+describe("University can see Registration Keys after subscribing", () => {
+
+  beforeEach(() => {
+    cy.server();
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v0/articles",
+      response: "fixture:articles.json",
+      status: 200
+    });
+  });
+
```

Comment:
you dont need to test for the articles to be displayed in this feature

Reply:
It's not that the purpose it to test the articles being displayed, but when you start the server the first component rendered is the homepage, that will always look for articles to display, and if you don't define the route with the fixture with the stubbed response, we will get an aborted request in the test, and we don't want that